Examples/Widgets/HospitalBedCapacity/src/components/HospitalWidget.ts

```diff
@@ -38,8 +42,8 @@ export default class HospitalWidget extends LitElement {
    */
   @property({ type: String, reflect: true, attribute: "covi-api-key" })
   covidApiKey = "";
-  @property({ type: Number, reflect: true }) latitude = 37.40527;
-  @property({ type: Number, reflect: true }) longitude = -122.01178;
+  @property({ type: Number, reflect: true }) latitude = 0;
```

Comment:
same here i guess. although i would expect these to be passed from index.ts

Reply:
I believe it would be redundant if added here as well for that reason exactly.